### DISCUSSION TITLE
require statement left in bundle

### DISCUSSION THREAD

**Discussion by andrewalderson** [webpack/webpack#18231] [Q&A]
# Bug report
I originally filed this in the [Angular Cli repo](https://github.com/angular/angular-cli/issues/26200#issue-1973383012). I am having an issue where a ```require``` statement is being left in the compiled code. The code is running in a Web Worker and I am not sure if that is an issue.

**What is the current behavior?**
The ```require``` statement is left in the code and causing an error

**If the current behavior is a bug, please provide the steps to reproduce.**

[Stackblitz](https://stackblitz.com/edit/stackblitz-starters-fnwhlu)

If you open the console for the Stackblitz sample you will see the error pointing to the ```require``` statement in the code. This is also not inlined in the production build.

**What is the expected behavior?**

The ```require``` statement should be properly handled.

**Other relevant information:**
webpack version:  Angular Cli is using 5.89.0
Node.js version: 18
Operating System: Linux (not sure what the Stackblitz container is running) but I was running this in a Linux devcontainer locally
Additional tools: Angular Cli

**Comment by alexander-akait** [comment]
@alan-agius4 There are no webpack problems, something insert `require` after webpack finished. Do you run babel after webpack generates output (i.e. lower async syntax)?
